content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -222,6 +227,83 @@ At this point, the flow is now complete.
    order to confirm the delivery. Otherwise, the delivered quantity can be entered manually directly
    on the sales order.
 
+.. _sales/invoicing/down_payment/100-percent-down-payments:
+
+Example: request 100% down payment
+==================================
+
+The process of requesting a 100% down payment is similar to the process of setting up a :ref:`50%
+down payment <sales/invoicing/down_payment/50-percent-down-payments>`, but with fewer steps.
+
+.. note::
+   A 100% down payment is **not** the same as a full payment of the sales order.
+
+   A sales order paid through the regular invoice process will not allow any additional invoices to
+   be generated, and **will not** display the *Create invoice* button on the Sales Order.
+
+   Following this example **will** cause the *Create invoice* button to be displayed on the Sales
+   Order. This is because Odoo expects another invoice to be created after the down payment
+   to complete payment of the sales order.
+
+
+The *Solar Panel Installation* product is being used in this example.
+
+To configure a 100% down payment, begin by navigating to :menuselection:`Sales app --> New`, and add
+a :guilabel:`Customer` to the quote.
+
+Next, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
+:guilabel:`Solar Panel Installation` product.
+
+Upon clicking the :guilabel:`Confirm` button, the quotation turns into a sales order. At that point,
+an invoice can now be created by clicking :guilabel:`Create Invoice` in the top-left corner.
+
+.. image:: down_payment/100p-order-confirmed.png
+   :align: center
+   :alt: Confirmed sales order for the Solar Panel Installation product.
+
```

Comment:
To try and cut down on the number of images in this doc, I think we can remove most of these. Especially since the process is so similar to the 50% DP example, and your written instructions are clear. If you feel like these images are important to keep, please feel free to do so 👍
```suggestion
```